src/Exceptions/IntercomApiException.php

```
@@ -0,0 +1,53 @@
+    public function getBody(): mixed
+    {
+        return $this->body;
+    }
+
+    /**
+     * @return string
+     */
+    public function __toString(): string
+    {
+        if (empty($this->body)) {
+            return "$this->message; Status Code: $this->code\n";
```

PHPStan (2.1.17)

Part $this->code (mixed) of encapsed string cannot be cast to string.


(encapsedStringPart.nonString)